content/react-lsn1/exercises/_index.md

```diff
@@ -0,0 +1,222 @@
+---
+title: "Exercises: Chores vs. Hobbies"
+date: 2023-03-31T09:42:17-05:00
+draft: false
+weight: 2
+originalAuthor: Sally Steuterman # to be set by page creator
+originalAuthorGitHub: gildedgardenia # to be set by page creator
+reviewer: Kimberly Horan # to be set by the page reviewer
+reviewerGitHub: codinglikeagirl42 # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+The following exercises walk you through the process of modifying existing
+components and creating new ones.
+
+## Starter Code
+
+For this set of exercises, you will need a copy of the LaunchCode Angular
+projects repository. If you haven't done so already,
+
+1. Fork the [react-exercises-and-studios repository](https://github.com/LaunchCodeEducation/react-exercises-and-studios)
+   on GitHub.
+1. Clone your fork.
+
+Navigate into the exercises now:
+
+```console
+   $ pwd
+      react-projects
+   $ ls
+      part1 part2 part3
+   $ cd part1
+   $ cd exercises
+```
+
+You should be able to run  `npm start`, but the page will not look pretty. Time to fix that!  
```

Comment:
Add a note about using npm install, to install the dependencies.

Reply:
Done! Thank you!